back-end/API/app/controllers/jobOffer_controller.py

```
@@ -154,23 +154,25 @@ def offresEmploiApproved():
+        if data.get('isApproved'):
+            user_service.manageTemporaryEnterprise(jobOfferToUpdate, selectedEnterpriseId)
+
         jobOffer_service.approveJobOffer(id, data['isApproved'], data['approbationMessage'])
-        
-        print(jobOfferToUpdate)
-        current_user_job = User.query.filter_by(id=jobOfferToUpdate.last_modified_by_id ).first()
-        print(current_user_job)
+
+        current_user_job = User.query.filter_by(id=jobOfferToUpdate.last_modified_by_id).first()
 
         if data['isApproved'] == True:
```

Ruff (0.15.9)

[error] Avoid equality comparisons to `True`; use `data['isApproved']:` for truth checks

Replace with `data['isApproved']`

(E712)